packages/cli/src/commands/bundle/coerces/index.ts

```diff
@@ -0,0 +1,59 @@
+import { readFileSync } from 'fs';
+import { GITHUB_ORG, DEFAULT_BRANCH, Repository } from '@bundler/core';
+import { RepositoryId } from '@bundler/github';
+import { CheckError } from '../../../common/errors';
+import { isRepoValid } from '../../../validation/formats';
+import { INPUT_BUNDLE_REQUEST_SCHEMA } from '../../../validation/schemas';
+import { validate } from '../../../validation/validator';
+import { CoerceFunc } from '../../../wrappers/coerce';
+import { InputFileBundleRequest } from '../bundleFactory';
+import { NAME_TO_REF_DELIMITER, OWNER_TO_NAME_DELIMITER } from '../constants';
+
+const repoStrToRepoId = (repo: string): RepositoryId => {
+  const [ownerAndName, ref] = repo.split(NAME_TO_REF_DELIMITER);
+  const [name, owner] = ownerAndName.split(OWNER_TO_NAME_DELIMITER).reverse();
+  return { owner: (owner as string | undefined) ?? GITHUB_ORG, name, ref: (ref as string | undefined) ?? DEFAULT_BRANCH };
```

Comment:
this line is a bit hard to read

```suggestion
return {
    name,
    owner: (owner as string | undefined) ?? GITHUB_ORG,
    ref: (ref as string | undefined) ?? DEFAULT_BRANCH
};
```

much clearer.

Reply:
done.